articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+ title: Blockchain as a Revolution in Health
+ description: Learn how Blockchain technology can provide a greater value of care and better health outcomes for patients.
+ author: jethro-magaji
+ date: 2020-08-24T00:00:00-10:00
+ topics: [Blockchain][Cryptocurrency][Healthtech]
+ excerpt_separator: <!--more-->
+ images:
+
+   - url: /engineering-education/blockchain-as-a-revolution-inhealthcare/Blockchain-Health-img.jpg
+     alt: Blockchain Health
+ ---
+ 
+[*Image Source: https://www.gettyimages.com/detail/illustration/caduceus-icon-on-money-and-cryptocurrency-royalty-free-illustration/920245220*](https://www.gettyimages.com/detail/illustration/caduceus-icon-on-money-and-cryptocurrency-royalty-free-illustration/920245220)
+
+# Blockchain as a Revolution in Healthcare
+![Blockchain img](/engineering-education/blockchain-as-a-revolution-inhealthcare/Blockchain-img.jpg)
+[*Image Source: https://www.gettyimages.com/detail/illustration/digital-room-with-padlock-and-word-royalty-free-illustration/748345923?adppopup=true*](https://www.gettyimages.com/detail/illustration/digital-room-with-padlock-and-word-royalty-free-illustration/748345923)
+
+[Blockchain represents great promise for stakeholders in healthcare fields](https://nimbus-t.com/blockchain-opportunities-for-health-care-deloitte-us). Using this technology has the potential to increase the standard of care. In the long run, using a [blockchain network nationwide](https://www.parathon.com/understanding-blockchain-in-healthcare-and-how-it-is-beneficial-to-providers.html) for electronic medical records (EMR) could help achieve better health outcomes for patients and improve efficiencies in healthcare delivery.
+
+## What is Blockchain?
+A Blockchain [can be defined as a continuously growing list of records, or blocks, which use cryptography to link and secure the individual blocks](https://nodemads.net/service/blockchaineducation). It can also be described as "an open, distributed ledger where transactions record can be verified between two parties and stored permanently," according to Nodemads. Blockchain is a distributed computing system which is secured by design. This makes Blockchain suitable for the performing task such as recording medical records and other management activities, including transaction processing, documenting provenance, voting, identity management, and food traceability. 
+
+## Advantages of Blockchain in Healthcare 
+***Provenance & Increased Data Integrity:*** [Transactions in the Blockchain network are immutable](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561). This means that once a transaction is approved, it cannot be altered or deleted. Transactions are shared across the network and securely recorded. The data integrity is safe, and time-stamp can be verified by any user. The original author of a transaction in a blockchain network is recorded and remains accessible for the entire life of the blockchain.
```

Comment:
After cannot be altered or deleted, maybe include one sentence on how this technology creates immutable records, for context.

Reply:
Done